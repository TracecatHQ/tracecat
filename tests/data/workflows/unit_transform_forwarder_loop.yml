title: Forwarder loop
description: Test that we can loop
entrypoint: a
inputs:
  first: 1
  second: 2
  list: [1, 2, 3]

actions:
  - ref: a
    action: core.transform.forward
    for_each: ${{ for var.x in INPUTS.list }}
    args:
      value: I received ${{ var.x }} from you
