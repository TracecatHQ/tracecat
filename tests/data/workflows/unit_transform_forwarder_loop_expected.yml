ACTIONS:
  a:
    result:
      - I received 1 from you
      - I received 2 from you
      - I received 3 from you
    result_typename: "list"

INPUTS:
  first: 1
  second: 2
  list: [1, 2, 3]
TRIGGER:
